templates/vhost/_php_admin.epp

```diff
@@ -0,0 +1,12 @@
+<% if $php_admin_values and !$php_admin_values.empty { -%>
+  <%- $php_admin_values.stdlib::sort_by |$m| { $m[0] }.each |$key, $value| { -%>
+  php_admin_value <%= $key %> <%= $value %>
+  <%- } -%>
+<% } -%>
+<% if $php_admin_flags and !$php_admin_flags.empty { -%>
+  <%- $php_admin_flags.stdlib::sort_by |$m| { $m[0] }.each |$key, $flag| { -%>
+    <%# normalize flag -%>
+    <%- if $flag.downcase =~ /true|yes|on|1/ { $_flag = 'on' } else { $_flag = 'off' } -%>
```

Comment:
Given the repetition, perhaps a helper would be useful? There's a lot of overlap with `$_php.epp` anyway so other comments I had there apply here too.